integration/simple_test.go

```diff
@@ -161,6 +161,57 @@ func (s *SimpleSuite) TestRequestAcceptGraceTimeout(c *check.C) {
 	}
 }
 
+func (s *SimpleSuite) TestCustomPingTerminationStatusCode(c *check.C) {
+	s.createComposeProject(c, "reqacceptgrace")
```

Comment:
Instead of using compose for this, since you are never actually checking the response, could you use a non-docker test server like here:

https://github.com/containous/traefik/blob/master/integration/headers_test.go#L38

This would make the test faster, since it does not require docker, and would make the code cleaner.